watcher_fsevents_cgo.go

```diff
@@ -63,6 +64,10 @@ var (
 func init() {
 	wg.Add(1)
 	go func() {
+		// There is exactly one run loop per thread. Lock this goroutine to its
+		// thread to ensure that it's not rescheduled on a different thread while
+		// setting up the run loop.
+		runtime.LockOSThread()
```

Comment:
Could we have `defer runtime.UnlockOSThread()`?

I agree it's not needed since the panic would crash running program - just for readability purposes.

Reply:
It's misleading to add this call if it never executes. We can still add it to waste some stack for aesthetic reasons ;)